schema/defs.json

```diff
@@ -41,10 +41,10 @@
             "minimum": 0,
             "maximum": 18446744073709552000
         },
-        "percent": {
+        "maximum": {
             "type": "integer",
             "minimum": 0,
```

Comment:
> Looking at this again. I think this is actually a generic percent definition, rather than specific to CPU Percent…

Ah, right.  But it looks like the Windows field is the sole consumer.  I'd recommend dropping `percent` from `defs.json` and inlining the new limits in the `maximum` entry in `config-windows.json`.  But I'm not a maintainer, so you may want to wait for one of them to chime in.